boards/nxp/mimxrt1180_evk/Kconfig

```diff
@@ -0,0 +1,6 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_MIMXRT1180_EVK_NETC_INIT_PRIO
```

Comment:
this wont be needed if using the board init hook

Reply:
okay, thanks.